plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/active-filters/types.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * External dependencies
+ */
+import { BlockEditProps } from '@wordpress/blocks';
+
+export type BlockAttributes = {
```

Comment:
If we don't use attribute outside of the edit component, then we don't need this file. But usually, we need to use block attributes and/or edit props in other components, then a separate file makes more sense to me. Because it's something happy more often than not, I think following the 'convention' (even when the attribute is used only in one place) make it easier for every one working on the source.